app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  has_many :groups, dependent: :destroy
+  has_many :entities, dependent: :destroy
+  has_one_attached :avatar, dependent: :destroy
+  validates :name, presence: true, uniqueness: true
```

Comment:
- [ ] Great job using the devise for authentication however you need to validate all user input  (email, password) to make sure that anyone with bad intentions cannot compromise their app. Please add this 👍🏻

Reply:
Solved thank you